docs/topics/Gateway.md

```diff
@@ -53,7 +53,7 @@ The Discord Gateway has a versioning system which is separate from the core APIs
 | Field | Type | Description | Present |
 |-------|------|-------------|---------|
 | op | integer | opcode for the payload | Always |
-| d | ?mixed (object, integer, bool) | event data | Always |
+| d | ?mixed (object, integer, bool, list) | event data | Always |
```

Comment:
Make this `array` instead for consistency of other docs

Reply:
The dapi docs uses list.